### PR TITLE
Small rating table fixes

### PR DESCRIPTION
**Pull request by IidaKainu**
Render newlines and set max height higher

### CODE REVIEW

**Comment by juhoeerola** [comment]
I noticed that there seems to be no limit for the height of the comment box.
Now that newlines are rendered the box is more likely to grow "too much" vertically, so maybe we could limit it a bit.

**Comment by IidaKainu** [comment]
Added a limit, hopefully it is not too restricting.

File: frontend/src/components/RatingTable.module.scss [hunk 1]
```diff
@@ -72,14 +72,23 @@
...
   color: $COLOR_BLACK60;
+
   padding: 15px 20px;
   margin-top: 15px;
   border-radius: 10px;
+
   text-align: start;
+  white-space: pre-line;
+  max-height: 250px;
+  overflow: hidden;
```
Comment (juhoeerola) [inline frontend/src/components/RatingTable.module.scss:85]: > hopefully it is not too restricting

Why not make it scrollable?

```suggestion
  overflow-y: auto;
```

**Review by juhoeerola — CHANGES_REQUESTED** [review]

File: frontend/src/components/RatingTable.module.scss (hunk 1 above)
Comment (IidaKainu) [reply frontend/src/components/RatingTable.module.scss]: Good point

**Review by juhoeerola — APPROVED** [review]
👍

**Pull request merged**